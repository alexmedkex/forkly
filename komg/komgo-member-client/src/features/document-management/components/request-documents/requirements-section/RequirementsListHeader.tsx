import * as React from 'react'
import styled from 'styled-components'

export const RequirementsListHeader = () => {
  return (
    <HeaderContainer data-test-id="requirements-section-header">
      <p>Document Type</p>
      <p>Requirements</p>
      <p>Attachment</p>
    </HeaderContainer>
  )
}

const HeaderContainer = styled.div`
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  border-bottom: 1px solid #c0cfde;
  text-transform: uppercase;
  line-height: 21px;
  font-size: 11px;
  font-weight: 600;
  color: #5d768f;
  margin: 16px 0;
`
