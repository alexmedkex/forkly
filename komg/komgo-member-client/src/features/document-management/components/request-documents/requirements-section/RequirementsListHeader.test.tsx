import * as React from 'react'
import { RequirementsListHeader } from './RequirementsListHeader'
import * as renderer from 'react-test-renderer'

describe('RequirementsListHeader component', () => {
  it('renders', () => {
    expect(renderer.create(<RequirementsListHeader />).toJSON()).toMatchSnapshot()
  })
})
