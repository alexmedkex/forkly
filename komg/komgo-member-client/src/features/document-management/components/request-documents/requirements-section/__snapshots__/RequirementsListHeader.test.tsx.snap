// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RequirementsListHeader component renders 1`] = `
.c0 {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  border-bottom: 1px solid #c0cfde;
  text-transform: uppercase;
  line-height: 21px;
  font-size: 11px;
  font-weight: 600;
  color: #5d768f;
  margin: 16px 0;
}

<div
  className="c0"
  data-test-id="requirements-section-header"
>
  <p>
    Document Type
  </p>
  <p>
    Requirements
  </p>
  <p>
    Attachment
  </p>
</div>
`;
